pkg/sql/sem/tree/schedule.go

```diff
@@ -38,6 +39,11 @@ func (node *ScheduledBackup) Format(ctx *FmtCtx) {
 		ctx.FormatNode(node.ScheduleLabel)
 	}
 
+	if node.IfNotExists ==true{
```

Comment:
nit: `if node.IfNotExists` is sufficient. Lets also move this above the schedule label once you have modified the `sql.y` rule.

Reply:
done